Assets/Script/ZoomCamera.cs

```diff
@@ -0,0 +1,35 @@
+using System.Collections;
```

Comment:
Why does this file exist ? 🤔 this logic only relates to the camera object you could add this functionality in the "camera controller"  in a method and reference the camera distance with `this.distance` or just `distance` WDYT ?